packages/twenty-front/src/utils/image/getImageAbsoluteURI.ts

```diff
@@ -1,15 +1,26 @@
 import { REACT_APP_SERVER_BASE_URL } from '~/config';
 
-export const getImageAbsoluteURI = (imageUrl?: string | null) => {
+type ImageAbsoluteURI<T extends string | null | undefined> = T extends string
+  ? string
+  : null;
+
+export const getImageAbsoluteURI = <T extends string | null | undefined>(
```

Comment:
Looks like ChatGPT stuff ;)

this looks over complex to me + it's a new pattern in the code base + it allows templating which doesn't seem to be the will here.

I think what you want is function overloading:
```
type ImageUri = string | null | undefined;

// Overload declarations
function getImageAbsoluteUrl(imageUri: string): string;
function getImageAbsoluteUrl(imageUri: null | undefined): null;

// Implementation
function getImageAbsoluteUrl(imageUri: ImageUri): string | null {
  return typeof imageUri === 'string' ? imageUri : null;
}

const myReturn = getImageAbsoluteUrl('toto'); // Type of myReturn is `string`
const myReturn2 = getImageAbsoluteUrl(null); // Type of myReturn2 is `null`
const myReturn3 = getImageAbsoluteUrl(undefined); // Type of myReturn3 is `null`
```